certified-connectors/Azure Communication Services Email/apiDefinition.swagger.json

```diff
@@ -173,15 +173,6 @@
           }
         ],
         "responses": {
-          "default": {
```

Comment:
It appeared that 202 response is ignored, and in a flow, the default response is always shown as the output of this action. So we decided to remove this "default" response instead to force the 202 response to surface as the action outputs. Is there another way around this?

Reply:
It failed the change validation...

Reply:
@lei-msft @alvinjiang-msft I have checked the swagger and change validation logs before approving and merging your PR. 

`The response header has been removed. Existing flows may be broken if they are using this header and actual payload has been changed. ValuePath: operation/SendEmail/responses/default/headers/x-ms-error-code Source: NewSwagger`

This is usually a breaking change, but your connector is never published into any production region and we allow breaking changes for brand new connectors. Please let me know if you have any questions.

Reply:
In that case, this particular `Change Validation` failure is expected, and we can merge it. Thx @sriyen-msft!